components/automate-ui/src/app/ngrx.reducers.ts

```diff
@@ -170,7 +172,8 @@ export const defaultInitialState = {
   serviceGroups: serviceGroups.ServiceGroupEntityInitialState,
   teams: teamEntity.TeamEntityInitialState,
   userperms: permEntity.initialState,
-  users: userEntity.UserEntityInitialState
+  users: userEntity.UserEntityInitialState,
+  userself: userSelfEntity.UserSelfEntityInitialState
```

Comment:
should this be camel cased like below?

Reply:
I don't know. It should be one way or the other though.

Reply:
I think you are right. I will switch to "userSelf".

Reply:
Good catch! I am not sure why that did not break things.

Reply:
hrm ya i would have expected something to break. oh well 🤷‍♀